src/odemis/util/test/dataio_test.py

```diff
@@ -223,6 +223,60 @@ def test_splitext(self):
             self.assertEqual(ao, eo, "Unexpected output for '%s': %s" % (inp, ao))
 
 
+class TestSplitPlanes(unittest.TestCase):
+    def _input_image_2d(self):
+        metadata = {model.MD_SW_VERSION: "1.0-test",
+                    model.MD_HW_NAME: "fake hw",
+                    model.MD_DESCRIPTION: "sem",
+                    model.MD_DIMS: "XY"}
+
+        return model.DataArray(numpy.zeros((512, 512), dtype=numpy.dtype("uint16")), metadata.copy())
+
+    def _input_image_3d(self):
+        metadata = {model.MD_SW_VERSION: "1.0-test",
+                    model.MD_HW_NAME: "fake hw",
+                    model.MD_DESCRIPTION: "sem",
+                    model.MD_DIMS: "CTZYX",
+                    }
+
+        return model.DataArray(numpy.zeros((3, 8, 10, 256, 512), dtype=numpy.dtype("uint16")), metadata.copy())
+
```

Comment:
Should there not also be a test case for when the MD_DIMS <= 2 and not "XY" or "YX" since it should also directly return the data when `len(dims) <=2`.